nixos/modules/virtualisation/lxd.nix

```diff
@@ -69,8 +81,9 @@ in
         ExecStart = "@${pkgs.lxd.bin}/bin/lxd lxd --group lxd";
         Type = "simple";
         KillMode = "process"; # when stopping, leave the containers alone
+        LimitMEMLOCK = mkIf cfg.productionSetup "infinity";
+        LimitNOFILE = mkIf cfg.productionSetup "1048576";
```

Comment:
I think this service settings should always be added (not in production only). The debian lxd packages seem to do that too https://github.com/lxc/lxd-pkg-ubuntu/blob/dpm-bionic/debian/lxd.service#L15.

Reply:
Good point. I've added some stuff from https://github.com/lxc/lxd-pkg-ubuntu/blob/dpm-xenial/debian/lxd.service , how does this look?

Reply:
Looks great! I would only rename the option from `productionSetup` to `recommendedSysctlSettings` in the style of the `recommended*` nginx options in nixos: https://nixos.org/nixos/options.html#nginx.reco

Reply:
Fixed.